dss/api/files.py

```diff
@@ -96,6 +97,11 @@ def get_helper(uuid: str, replica: typing.Optional[str]=None, version: str=None)
 
 @dss_handler
 def put(uuid: str, extras: dict, version: str=None):
+    class CopyMode(Enum):
+        NO_COPY = auto()
+        COPY_INLINE = auto()
+        COPY_ASYNC = auto()
```

Comment:
You're not a fan of the more concise `CopyMode = Enum("CopyMode", "NO_COPY COPY_INLINE COPY_ASYNC")?